app-template/config-template.xml

```diff
@@ -95,6 +95,7 @@
       <variable name="THIRD_URL_SCHEME" value="bitcoincash" />
       <variable name="FOURTH_URL_SCHEME" value="bchtest" />
       <variable name="FIFTH_URL_SCHEME" value="ethereum" />
+      <variable name="SIXTH_URL_SCHEME" value="ripple" />
```

Comment:
You need to update Custom-URL-scheme version to v8.0.0

Reply:
Thanks! Updated in `package.json`